content/docs/command-reference/exp/run.md

```diff
@@ -118,6 +118,12 @@ changes between/after queueing runs.
 `dvc exp run --temp`, for example to continue working on the project meanwhile
 (e.g. on another terminal).
 
+> ⚠️ Note that only tracked files and directories will be included in
+> `--queue/temp` experiments. To include untracked files, stage them with
+> `git add -N` first (before `dvc exp run`). Feel free to `git reset` them
+> afterwards. Git-ignored files/dirs are explicitly excluded from runs outside
+> the workspace to avoid committing secrets into experiments.
```

Comment:
```suggestion
> the workspace to avoid committing unwanted files into experiments.
```
I wouldn't restrict it to secrets. It could also be lots of extraneous files, like the venv scenario.